Source/Core/VideoBackends/Software/Clipper.cpp

```diff
@@ -389,6 +389,7 @@ namespace Clipper
 			}
 
 			OutputVertexData triangle[3];
```

Comment:
Yes (value initialization that resolves to zero-initialization because of the lack of said constructor).